tests/integration/update.bats

```diff
@@ -284,7 +284,7 @@ function check_cpu_shares() {
 @test "update cgroup cpu limits" {
     [[ "$ROOTLESS" -ne 0 ]] && requires rootless_cgroup
 
-    # run a few busyboxes detached
+    # run a few containeres detached
```

Comment:
nit: `s/containeres/containers/`

In fact maybe comments like this just need to be dropped, as the code (`runc run -d ...`) is obvious and does not really call to be commented.

Reply:
I took a stab at it and removed most of the references.

Looking back, I think this is probably a broader topic that should be tackled outside this PR. A lot of the comments are very similar in that they convey only the intent of the next line which is usually pretty obvious:
```
# Start the command.
runc start test_container

# Kill the container.
runc kill test_container KILL

# check state
testcontainer test_container running

# test state of container is paused
testcontainer test_container paused
```

Overall this seems like a discussion that's pretty close to a coding style discussion and might have a better place in an issue first so that there's consensus on how this should be addressed :) 

What do you think?

Reply:
> I took a stab at it and removed most of the references.

Thanks!

> What do you think?

I agree these comments do not add any value and should probably be removed. I also agree this is out of scope for this PR.